hydra-node/exe/tx-cost/TxCost.hs

```diff
@@ -133,25 +112,25 @@ computeCommitCost =
           catMaybes
             <$> forM
               valueSizes
-              ( \sz -> do
-                  utxo <-
-                    generate $
-                      genSimpleUTxOOfSize numUTxO
-                        >>= resizeValuesTo sz
-                  (commitTx, knownUtxo) <- generate $ genCommitTx utxo
-                  case commitTx of
-                    Left _ -> pure Nothing
-                    Right tx -> do
-                      let txSize = LBS.length $ serialize tx
-                      if txSize < fromIntegral (Ledger._maxTxSize pparams)
-                        then case evaluateTx tx (utxo <> knownUtxo) of
-                          (Right (toList -> [Right (Ledger.ExUnits mem cpu)])) ->
-                            pure $ Just (NumUTxO $ length utxo, ValueSize sz, TxSize txSize, MemUnit mem, CpuUnit cpu)
-                          _ -> pure Nothing
-                        else pure Nothing
-              )
+              (compute numUTxO)
       )
  where
+  compute numUTxO sz = do
+    -- FIXME: genSimpleUTxOOfSize only produces ada-only, so the separation of
+    -- generating and resizing is moot
```

Comment:
@abailly-iohk what do you think about this? Shall we move to just `ada-only` utxos and simplify the results by having a single dimension?

Reply:
Yeah, makes a lot of sense, let's do that